homeassistant/components/arwn/sensor.py

```diff
@@ -94,12 +94,16 @@ def async_sensor_event_received(msg):
                 sensor.hass = hass
                 sensor.set_event(event)
                 store[sensor.name] = sensor
-                _LOGGER.debug(
-                    "Registering new sensor %(name)s => %(event)s",
+                _LOGGER.info(
```

Comment:
I don't think that we should do this. It's not info for normal runs. Instead users can use the `logger` integration to set individual package log levels.

Reply:
ack